learn/teams/manage-projects.md

```diff
@@ -18,29 +18,47 @@ Members should be registered with **Teams** before they can be invited to a proj
 
 You can access the Manage Projects page in the administration area by clicking **Manage Projects** from the left panel.
 
-![Manage projects](/learn/assets/TeamManageProjects.png)
+![Manage projects](/learn/assets/wm_restructureMP_1_f.png)
 
 From the projects list, you can
 
+- Import project from code repository
+- Export project details
 - Add user to a project  
-- View or edit a user’s role in a project  
-- View project information  
-- Remove a user from a project
+- View or edit user’s role in a project  
+- Add and view project information  
+- Remove a user from project
+
+## Import Project from Code Repository
+
+You can import a project available on Version Control Systems (VCS) like GitHub, GitLab, Bitbucket and Azure. It also imports commit history and specified branches of the project.

```

Comment:
Rewrite

It also imports should be replaced with. Commit history of the specified branch carries over when imported.